src/Infrastructure/Services/UserService.cs

```diff
@@ -183,6 +188,28 @@ public async Task<Response<NhsLoginResponse>> NhsLoginAsync(string code, string
             }
         }
 
+        private async Task SendContentfulEmailAsync(string emailName, string emailRecipient, CultureInfo selectedLocale,
+            string firstName = null)
+        {
+            var contentfulEmail = await _contentfulService.GetContentfulEmailAsync(emailName, selectedLocale);
+            string htmlContent = _richTextToHtmlConverter.Convert(contentfulEmail.EmailBody);
+
+            if (!string.IsNullOrEmpty(firstName))
+            {
+                var template = Handlebars.Compile(htmlContent);
+                var data = new
+                {
+                    firstName =
+                        selectedLocale.TextInfo.ToTitleCase(firstName.ToLower()),
```

Comment:
No need to call `ToLower()` here as that part is not locale specific and we are shifting it into TitleCase anyway.

Reply:
I tried removing the ToLower and it sent through the firstName as all Caps so I had to add it again to work

Reply:
Is it coming through originally in all caps? That might be why, it is being interpreted as an abbreviation.

In that case the `ToLower()` has to be locale aware.

`selectedLocale.TextInfo.ToTitleCase(firstName.ToLower(selectedLocale)))`

Reply:
Yes all caps.  I will update

Reply:
why does the `ToLower()` need to be locale aware?  And the title case for that matter?

Reply:
In isolation the characters are just unicode symbols, they don't mean anything except when put into the context of a locale. Different locales will have different rules about what symbols are uppercase/lowercase and how you map between them, it might even be specific to where the character appears in the word or when it is next to a certain character.

So to make something lower case, you need to know that it is lowercase in English or lowercase in French (maybe there are rules for accents, etc.)

In this PR we want to change the case of the first name and rather than use the culture rules for the server we should use the culture rules of the email that we are constructing.

Some further reading:
https://haacked.com/archive/2012/07/05/turkish-i-problem-and-why-you-should-care.aspx/

That is also why when you want to make a conversion or comparison that always behaves the same regardless of what the user or server locale settings are you use the "invariant culture". This is useful for internal calculations and things you want to be consistent (even if they are technically wrong - at least it is not a weird bug that only happens when the server has it's locale set to en-US).

Anything that is user facing like these emails should be fully culture aware. Anything internal that some functionality depends on should default to using the InvariantCulture unless there is a good reason to do otherwise.

https://learn.microsoft.com/en-us/dotnet/api/system.globalization.cultureinfo.invariantculture?view=net-7.0